cyvcf2/cyvcf2.pyx

```diff
@@ -905,7 +906,7 @@ cdef class VCF(HTSFile):
             nv += 1
             if nv == n_variants:
                 break
-        sys.stderr.write("tested: %d variants out of %d\n" % (nv, nvt))
+        warnings.warn("tested: %d variants out of %d" % (nv, nvt))
```

Comment:
Not sure we want to keep this?

Reply:
yes, we can leave this. It's only used by peddy. I wouldn't do it this way again, but let's leave for now.